docs/extending/authentication.md

```diff
@@ -30,6 +30,14 @@ public interface IAccessTokenProvider
 }
 ```
 
+## Allowed hosts validator
+
+This utility class is in charge of validating the host of any request is present on an allow list. This is used by the base access token provider to check whether it should or not return a token to the request.
```

Comment:
Does this mean we should implement some `BaseAccessTokenProvider` to make the allowed hosts validation accessible to future access token providers?

Reply:
No, the goal of having a dedicated class is to enable composition over inherence